app/api/models/user.py

```diff
@@ -23,3 +26,40 @@ def add_models_to_namespace(api_namespace):
         ),
     },
 )
+
+login_request_body_model = Model(
+    "Login request data model",
+    {
+        "username": fields.String(required=True, description="User's username"),
+        "password": fields.String(required=True, description="User's password"),
+    },
+)
+
+login_response_body_model = Model(
+    "Login response data model",
+    {
+        "access_token": fields.String(required=True, description="User's access token"),
+        "expiry": fields.Float(
+            required=True, description="Access token expiry UNIX timestamp"
+        ),
+        "access_expiry": fields.Float(
+            required=True, description="Access token expiry UNIX timestamp"
+        ),
+        "refresh_token": fields.String(
```

Comment:
As we talked before, you don't need to thing about Refresh API for now, so these 2 attributes can be removed: `refresh_token` and `refresh_expiry`

Reply:
Got it. Will remove it now.